core/client/fs/src/main/java/alluxio/client/file/cache/store/MemoryPageStore.java

```diff
@@ -68,6 +80,7 @@ public void delete(PageId pageId) throws IOException, PageNotFoundException {
     if (!mPageStoreMap.containsKey(pageKey)) {
       throw new PageNotFoundException(pageId.getFileId() + "_" + pageId.getPageIndex());
     }
+    mPagePool.returnPage(mPageStoreMap.get(pageKey));
```

Comment:
nit: keep the api naming of the buffer pool align with others

Reply:
done